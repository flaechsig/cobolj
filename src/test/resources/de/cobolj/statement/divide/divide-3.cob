IDENTIFICATION DIVISION.
PROGRAM-ID. HELLO.

DATA DIVISION.
   WORKING-STORAGE SECTION.
   01 WS-NUMBER-RESULT-1 PIC 9(2) VALUE 51.
   01 WS-NUMBER-RESULT-2 PIC 9(1) VALUE 9.

PROCEDURE DIVISION.
   divide 2 INTO 111 GIVING WS-NUMBER-RESULT-1 WS-NUMBER-RESULT-2
   ON SIZE ERROR DISPLAY "KEIN SPEICHERUEBERLAUF".
   DISPLAY WS-NUMBER-RESULT-1.
   DISPLAY WS-NUMBER-RESULT-2.
STOP RUN.
