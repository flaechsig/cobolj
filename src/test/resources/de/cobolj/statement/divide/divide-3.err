Division eines Wertes auf mehrere Ziele (über Giving) mit Speicherüberlauf