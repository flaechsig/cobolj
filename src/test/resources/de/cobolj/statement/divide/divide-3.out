KEIN SPEICHERUEBERLAUF
55
9
